src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const tableBody = document.querySelector('table > tbody');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const container = document.querySelector('.container');
+
+container.addEventListener('click', e => {
+  const item = e.target.closest('.button');
+
+  if (!item) {
+    return;
+  }
+
+  const rows = document.querySelectorAll('table > tbody > tr');
+
+  const maxRow = 10;
+  const minRow = 2;
+  const maxColumn = 10;
+  const minColumn = 2;
+
+  switch (item) {
+    case appendRow:
+      tableBody.append(tableBody.children[0].cloneNode(true));
+
+      if (tableBody.children.length === maxRow) {
+        appendRow.disabled = true;
+      }
+
+      break;
+
```

Comment:
As for me, extra lines. Plus in style-guide doesn't have them: 
https://mate-academy.github.io/style-guides/javascript.html#154-for-more-information-seetruth-equality-and-javascriptby-angus-croll